src/jit/morph.cpp

```diff
@@ -11496,7 +11501,16 @@ GenTreePtr Compiler::fgMorphSmpOp(GenTreePtr tree, MorphAddrContext* mac)
                         }
                     }
                 }
-                fgMorphRecognizeBoxNullable(tree);
+
+                __fallthrough;
+
+            case GT_GT:
+
+                if ((oper != GT_GT) || ((tree->gtFlags & GTF_UNSIGNED) != 0))
```

Comment:
I prefer to keep the __fallthrough. Otherwise I'll have to duplicate the call as well as the two lines after it.